examples/blaze/src/main/scala/com/example/http4s/blaze/demo/server/service/GitHubService.scala

```diff
@@ -0,0 +1,55 @@
+package com.example.http4s.blaze.demo.server.service
+
+import cats.effect.Sync
+import cats.syntax.functor._
+import com.example.http4s.blaze.demo.server.endpoints.ApiVersion
+import fs2.Stream
+import io.circe.generic.auto._
+import org.http4s.circe._
+import org.http4s.client.Client
+import org.http4s.client.dsl.Http4sClientDsl
+import org.http4s.{Header, Request, Uri}
+
+// See: https://developer.github.com/apps/building-oauth-apps/authorization-options-for-oauth-apps/#web-application-flow
+class GitHubService[F[_]: Sync](client: Client[F]) extends Http4sClientDsl[F] {
+
+  // NEVER make this data public! This is just a demo!
+  private val ClientId     = "959ea01cd3065cad274a"
+  private val ClientSecret = "53901db46451977e6331432faa2616ba24bc2550"
```

Comment:
Whose data is this?

Reply:
It's mine. An example GitHub app I just created only for this example, so it's okay to make it public.